pbspro.spec.in

```diff
@@ -256,12 +254,6 @@ cd build
 %endif
 %if %{with cpuset}
 	--enable-cpuset \
-%endif
```

Comment:
why did you have to change the spec file?

Reply:
Looks like he was cleaning out the IBM legacy code. Assuming that's the case, I'd expect to see a similar change in pbspro.spec. You'll need to regenerate it by running autogen.sh.

Reply:
I was just moving code from the spec file to the spec.in file.  At the same time the changes from spec->spec.in was made, @subhasisb made changes to the spec file removing old code.  When configure ran, pbspro.spec would show up as modified since the checked in version was different from the generated version.  I'm just making them consistent.

Reply:
Thanks for explaining.